students/juyeob/westagram/users/views.py

```diff
@@ -1,10 +1,10 @@
 
-import json
-import re
+import json, bcrypt, jwt, re
 
 from django.http  import JsonResponse
 from json.decoder import JSONDecodeError
 from django.views import View
+from django.conf  import settings
```

Comment:
settings을 전부 불러오지 말고 필요한 것만 가져오세요.

```suggestion
from django.conf.settings  import SECRET_KEY
```

Reply:
작동이 안되서 검색해봤는데 레퍼런스 문서에서 이렇게 설명하고 있는데 어떻게 해야할가요?
- https://docs.djangoproject.com/en/4.0/topics/settings/

Reply:
settings.py랑 제가 헷갈렷네요. 
settings.py를 가져왔으면 제가 리뷰 드린 내용으로 하시면 되고, django.conf에서 가져온 settings object이면 원래대로 하시면 됩니다.
```suggestion
from django.conf  import settings
```